caluma/form/serializers.py

```diff
@@ -107,7 +120,7 @@ def update(self, instance, validated_data):
                 "Input questions are not the same as current form questions"
             )
 
-        for sort, question in enumerate(reversed(questions)):
+        for sort, question in enumerate(reversed(questions), start=1):
```

Comment:
Out of curiosity: Why do we `reverse()` the questions here? Why are they not in "sorted" order already?

Reply:
same as above sort flag is not exposed to user interface. API user simply sends a list which is ordered and this representation will need to be represented in the database.

Reply:
reverse is used that we do not have to figure out the largest sort id by a query which could be racy but can simply add new questions with sort order 0

Reply:
I understand why you start at 1 (for inserting later), but the reverse I still don't quite get

Reply:
there is a mutation AddFormQuestions which adds a single question at the end of the list, as we use descending sort order this can simply be set to zero and it will be safe.

Reply:
one more thing:
input list comes in ascending order but sort order is in descending. Hence reversed.

Reply:
Aah okay, thanks for explaining!